docs/AUTHENTICATION.md

```diff
@@ -35,48 +35,45 @@ These are some of the terms that you will encounter in this document and other r
 
 The overall steps for this process are as follows:
 1. Obtain the needed configuration information, including Tenant ID, Client ID, and Client Secret
-1. Check the token (authentication) endpoint from the ADH discovery URL
+1. Check the token (authentication) endpoint from the CDS discovery URL
 1. POST the Client ID and Secret to the token endpoint in order to get an access (bearer) token 
-1. Pass it back to ADH in the Authorization header in subsequent calls (the base tenant endpoint is used in these samples)
+1. Pass it back to CDS in the Authorization header in subsequent calls (the base tenant endpoint is used in these samples)
+
+![image](https://github.com/user-attachments/assets/81dd8676-4fce-4b12-8a13-bc97e1dafbdb)
 
 The samples for this authentication flow are found in the `Client Credential Flow` section of the table below.
 
-### Hybrid Flow
+### Authorization Code Flow with PKCE
+
+If you are developing any application where a user (resource owner) needs to access resources, use this flow. This flow adds an extra layer of security on top of the OIDC implicit flow by:
 
-If you are developing a traditional thick applications, with the user trying to access resources this should be your authentication method of choice. This flow provides a more secure means of authenticating, but also has the most constraining requirements. In this case the user will authenticate against the identity provider using any type of browser, even an unsafe one. Once this is completed the server-side client will authenticate against the authorization server and be awarded an Access Token and a Refresh Token, which should never be displayed to the user or the browser.
+1. Requiring a client-verified code exchange for access token
+2. By not returning the access token in a redirect URI.
 
-The sample for this authentication flow can be found [here](https://github.com/osisoft/sample-adh-authentication_hybrid-dotnet).
+The client app uses its Client Id and Client Secret to request an authorization code from the authorization server. The user is then directed to authenticate themselves with the authorization server (login prompt, 2 factor authentication, etc.) If the user successfully authenticates, an authorization code is returned to the client app, and the client can request Access Tokens with this authorization code. The client can then pass this Access Token in the authorization header in requests to get data from CDS.
+
+![image](https://github.com/user-attachments/assets/f22457fb-0d46-4ff7-8af2-31acbb145464)
+
+In this flow, no refresh token is provided. Authorization Code Flow supports silent refresh, which makes it possible to receive a new access token while the user is both using the application and logged in with the Identity Provider in the same browser session. This is done behind the scenes without interrupting the user experience. 
```

Comment:
`Authorization Code Flow` is probably okay since it references the section header, but `Identity Provider` should be lower-cased